rtl/eclrtl/rtlformat.cpp

```diff
@@ -1546,7 +1546,7 @@ void CommonCSVWriter::addStringField(unsigned len, const char* field, const char
     v.append(csvQuote);
     escapeQuoted(len, field, v);
```

Comment:
Would a better fix be to convert \0 to a better ascii representation.  E.g. "\000" or similar?  Including null characters in csv text is dubious, and probably best not to directly include them in the result in case they cause other follow on problems.  What was the example that provoked the change (the Jira doesn't have any more details).

Reply:
It is a good idea to covert \0 to "\000". 
I just uploaded a ZAP report to the Jira. The ZAP report gives the example.